### ISSUE TITLE
Update VPC Connection create parameter 'Options'

### ISSUE THREAD

**Issue by chadell** [cloudtools/troposphere#923]
Reference: https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_CreateVpnConnection.html
"Options"
The options for the VPN connection.
Type: VpnConnectionOptionsSpecification object
Required: No
Example:
```
    Options={
        'StaticRoutesOnly': True|False,
        'TunnelOptions': [
            {
                'TunnelInsideCidr': 'string',
                'PreSharedKey': 'string'
            },
        ]
    }
```

**Comment by chadell** [comment]
Fix at #924

**Comment by chadell** [comment]
API and Cloudformation have different namings 😓
